sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/GetSchemasOperation.java

```diff
@@ -41,7 +41,7 @@
   .addStringColumn("TABLE_SCHEM", "Schema name.")
   .addStringColumn("TABLE_CATALOG", "Catalog name.");
 
-  private RowSet rowSet;
+  protected RowSet rowSet;
```

Comment:
Hm ........ but I noticed here is `o.a.hive` side code, @gatorsmile. Wouldn't it be better to avoid some changes here @gatorsmile to prepare to get rid of Hive stuff later completely?

Reply:
We do not have a plan to remove the thrift-server.

Reply:
https://github.com/apache/spark/commit/7feeb82cb7f462e44f7e698c7c3b6ac3a77aade4 shows we want to further cleanup and improve the thrift-server. Even if https://issues.apache.org/jira/browse/HIVE-16391 is resolved, we will still keep the Hive thrift-server.